src/api.cpp

```diff
@@ -2858,6 +2858,7 @@ cl_int CLVK_API_CALL clGetKernelInfo(cl_kernel kern, cl_kernel_info param_name,
     cl_int ret = CL_SUCCESS;
     size_t ret_size = 0;
     const void* copy_ptr = nullptr;
+    // TODO could this be a union
```

Comment:
Yeah, maybe we can save a few tens of bytes of stack depending on what the compiler does :). This would apply to all the clGet*Info functions. Probably not worth losing sleep over...

Reply:
I removed this comment.